AutoCollection/HeartBeat.ts

```diff
@@ -0,0 +1,138 @@
+import os = require("os");
+import TelemetryClient = require("../Library/TelemetryClient");
+import Constants = require("../Declarations/Constants");
+import Util = require("../Library/Util");
+import Config = require("../Library/Config");
+import Context = require("../Library/Context");
+
+class HeartBeat {
+
+    public static INSTANCE: HeartBeat;
+
+    private static AIMS_URI = "http://169.254.169.254/metadata/instance/compute";
```

Comment:
Do these need to be available on the object or are they internal?
If only used internally then I would suggest making them const values outside of the class -- for better minification